stp/constraints.lisp

```diff
@@ -205,7 +204,8 @@
                                                                  GAS_CONST_G_COLD_ACCOUNT_ACCESS))
 (defun (call---gas-value-transfer-cost)          (*    (call---transfers-value)
                                                        GAS_CONST_G_CALL_VALUE))
-(defun (call---gas-new-account-cost)             (*    (-    1    EXISTS)
+(defun (call---gas-new-account-cost)             (*    IS_CALL
+                                                       (-    1    EXISTS)
```

Comment:
Among all CALL-type instructions only `CALL` may lead to an account being created. This fixes a bug where `CALLCODE` would sometimes cost an extra `25_000` gas despite no such surcharge applying.